Source/Service/Requests/CreateAuthenticationEvent.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  CreateAuthenticationEvent.swift
+//  Cotter
+//
+//  Created by Albert Purnama on 3/1/20.
+//
+
+import Foundation
+
+public struct CreateAuthenticationEvent: APIRequest {
+    public typealias Response = CotterEvent
+    
+    public var path: String {
+        return  "/event/create"
+    }
```

Comment:
```suggestion
    public let path: String = "/event/create"
```

Reply:
path may have custom parameters that need to be defined inside the init, so using `let` wouldn't be possible.

So I will leave this as is for consistency